src/style.css

```diff
@@ -0,0 +1,37 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  margin: 0 2px;
+}
+
+.stars--1 > .stars__star:nth-child(1),
+.stars--2 > .stars__star:nth-child(-n + 2),
+.stars--3 > .stars__star:nth-child(-n + 3),
+.stars--4 > .stars__star:nth-child(-n + 4),
+.stars--5 > .stars__star:nth-child(n) {
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  margin: 0 2px;
```

Comment:
you already have this styles in stars__star, delete them from here and left only unique ones